collections/ui/API.md

```diff
@@ -204,6 +204,34 @@ import { SplitButton } from '@dhis2/ui'
 |value|string|||Value associated with the button. Passed in object to onClick handler|
 |onClick|function||||
 
+### Calendar
+
+#### Usage
+
+**Note**: If possible, import the component from the main UI (`@dhis2/ui`) package.
+
+
+```js
+import { Calendar } from '@dhis2/ui'
+```
+
+
+#### Props
+
+|Name|Type|Default|Required|Description|
+|---|---|---|---|---|
+|calendar|any||*|the calendar to use such gregory, ethiopic, nepali - full supported list here: https://github.com/dhis2/multi-calendar-dates/blob/main/src/constants/calendars.ts|
+|cellSize|string|`'32px'`||the size of a single cell in the table forming the calendar|
+|date|string|||the current selected date in the format YYYY-MM-DD - if nothing provided, it will default to today|
+|defaultToToday|boolean|||whether to default to today's date or not|
```

Comment:
isn't this no longer relevant? (also the text on `date` immediately above saying that `it will default to today`)?